src/router/badge/plain.ts

```diff
@@ -1,12 +1,10 @@
 import { type Context } from 'jsr:@oak/oak/'
 import { badgePlain } from '~/src/badge/plain.ts'
+import { badgeProps } from '../badgeRoutes.ts'
```

Comment:
```suggestion
import type { TBadgeProps } from '../badgeRoutes.ts'
```
Я долго не мог понять что это за переменная такая badgeProps... Ну есть же договорённость как должны именоваться типы! То же касается импортов типов.